numpy/core/src/npysort/mergesort.c.src

```diff
@@ -246,8 +246,9 @@ mergesort0_@suff@(@type@ *pl, @type@ *pr, @type@ *pw, @type@ *vp, size_t len)
 
 
 int
-mergesort_@suff@(@type@ *start, npy_intp num, PyArrayObject *arr)
+mergesort_@suff@(void *start, npy_intp num, void *varr)
 {
+    PyArrayObject *arr = varr;;
```

Comment:
Two many `;` at end of line. This will cause problems on windows with will treat `;;` and an empty statement rather than a declaration.